.travis.yml

```diff
@@ -0,0 +1,129 @@
+language: c
+
+jobs:
+  include:
+    - name: "Python 3.6 on Xenial Linux"
+      os: linux
+      language: c
+      env: PYTHON_VERSION=3.6
+
+    - name: "Python 3.7 on Xenial Linux"
+      os: linux
+      language: c
+      env: PYTHON_VERSION=3.7
+
+    - name: "Python 3.8 on Xenial Linux"
+      os: linux
+      language: c
+      env: PYTHON_VERSION=3.8
+
+    - name: "Python 3.6 on OSX"
+      os: osx
+      language: generic
+      env: PYTHON_VERSION=3.6
+
+    - name: "Python 3.7 on OSX"
+      os: osx
+      language: generic
+      env: PYTHON_VERSION=3.7
+
+    - name: "Python 3.8 on OSX"
+      os: osx
+      language: generic
+      env: PYTHON_VERSION=3.8
+
+sudo: false
+
+before_install:
+  # Make Python scripts executable
+  - chmod +x tests/*
+  - chmod +x examples/Icequake_Iceland/*.py
```

Comment:
Necessary? Presumably not if the dike example works without this

Reply:
Nope, they're not executed. It's just python that needs permissions - so I'm surprised if `python tests/test_benchmarks.py` doesn't work without this being set.